content/vault/v1.20.x/content/docs/audit/index.mdx

```diff
@@ -4,228 +4,133 @@ page_title: Audit Devices
 
-## Enabling/Disabling audit devices
+In parallel, the contents of the response for a list operation are often of little value. Most responses contain only a "keys" field with a list of IDs. Some API endpoints (e.g. [`/identity/entity-alias/id`](/vault/api-docs/secret/identity/entity-alias#list-entity-aliases-by-id)) additionally return a "key\_info" field with some additional information about each list entry; even in this case, the information returned is usually not sensitive.
 
-When a Vault server is first initialized, no auditing is enabled. Audit
-devices must be enabled by a root user using `vault audit enable`.
+The [`elide_list_responses`](/vault/api-docs/system/audit#elide_list_responses) option replaces `keys` and `key_info` lists in audit log entries for responses to list requests with the number of objects they contain. This improves compatibility with audit devices and reduces the amount of audit data that is ingested, stored, and processed, while retaining some level of information about what was returned to the client.
 
-When enabling an audit device, options can be passed to it to configure it.
-For example, the command below enables the file audit device:
+Here is an example of an audit record with elided lists (formatted with extra whitespace, and with fields not relevant to the example omitted):
 
-```shell-session
-$ vault audit enable file file_path=/var/log/vault_audit.log
+```json
+{
+  "type": "response",
+  "request": {
+    "operation": "list"
+  },
+  "response": {
+    "data": {
+      "key_info": 4,
+      "keys": 4 // instead of ["a", "b", "c", "d"]
+    }
+  }
+}
 ```
 
-In the command above, we passed the "file_path" parameter to specify the path
-where the audit log will be written to. Each audit device has its own
-set of parameters. See the documentation to the left for more details.
+## Logging of request headers
 
-~> Note: Audit device configuration is replicated to all nodes within a
-cluster by default, and to performance/DR secondaries for Vault Enterprise clusters.
-Before enabling an audit device, ensure that all nodes within the cluster(s)
-will be able to successfully log to the audit device to avoid Vault being
-blocked from serving requests.
-An audit device can be limited to only within the node's cluster with the [`local`](/vault/api-docs/system/audit#local) parameter.
+You can use the [`/sys/config/auditing`](/vault/api-docs/system/config-auditing#create-update-audit-request-header) API to configure which headers in incoming client requests Vault records in the corresponding audit log entries.
 
-When an audit device is disabled, it will stop receiving logs immediately.
-The existing logs that it did store are untouched.
+By default, Vault automatically records values for the User Agent header, as well as for the `correlation-id` and `x-correlation-id` request headers to assist with request correlation in distributed systems.
```

Comment:
```suggestion
By default, Vault records the following request headers to assist with request
correlation in distributed systems:

- `User-Agent`
- `Correlation-Id`
- `X-Correlation-Id`
```
Style correction: make important information easy to scan, write in active voice